python/ray/experimental/state.py

```diff
@@ -345,10 +348,22 @@ def task_profiles(self):
         executions of that task. The second element is a list of profiling
         information for tasks where the events have no task ID.
     """
+    if start == None:
+      start = 0
+    if end == None:
+      end = time.time()
+    if num == None:
+      num = sys.maxsize
```

Comment:
there is no need to do this because it looks like `None` is already the correct default argument for `zrangebyscore` https://redis-py.readthedocs.io/en/latest/#redis.StrictRedis.zrangebyscore.